# Kata 5
# Ejercicio 1: Utilizar operadores Aritméticos 
Exploremos cómo podemos crear un programa que pueda calcular la distancia entre dos planetas. Comenzaremos usando dos distancias de planetas: Tierra (149.597.870 km) y Júpiter (778.547.200 km).

In [9]:
tierra = 149597870 #km
jupiter = 778547200 #km

Realizar la operación
Con los valores obtenidos, es el momento de añadir el código para realizar la operación. Restarás el primer planeta del segundo para determinar la distancia en kilómetros. A continuación, puedes convertir la distancia del kilómetro en millas multiplicándola por 0.621.

In [21]:
diferencia = abs(tierra - jupiter) #absoluto por que no puede haber distancias negativas
conversion = round(diferencia * .621) #redondeado por que su valor es de .94
print(f"""La diferencia de los planetas en km es: {diferencia}
La diferencia de los planetas en millas es: {conversion}""")

La diferencia de los planetas en km es: 628949330
La diferencia de los planetas en millas es: 390577534


Con el código creado, ejecuta el notebook para obtener el resultado. Deberías recibir lo siguiente:

628949330

390577534

# Ejercicio 2: convierte cadenas en números y usa valores absolutos
Usando input, agrega el código para leer la distancia del sol para cada planeta, considerando 2 planetas.

In [25]:
pla1= input("Distancia al sol de planeta 1: ")
pla2= input("Distancia al sol de planeta 2: ")

Debido a que input devuelve valores de cadena, necesitamos convertirlos en números. Para nuestro ejemplo, usaremos int

In [26]:
val1 = int(pla1)
val2 = int(pla2)

Con los valores almacenados como números, ahora puedes agregar el código para realizar el cálculo, restando el primer planeta del segundo. Debido a que el segundo planeta podría ser un número mayor, usarás abs para convertirlo a un valor absoluto. También agregarás el código para mostrar el resultado en millas multiplicando la distancia del kilómetro por 0.621

In [27]:
diferencia = abs(val2 - val1)
conversion = diferencia * .621
print(f"""La diferencia de los planetas en km es: {diferencia}
La diferencia de los planetas en millas es: {conversion}""")

La diferencia de los planetas en km es: 4437200000
La diferencia de los planetas en millas es: 2755501200.0
